src/index.ts

```diff
@@ -613,6 +638,10 @@ function isChromeiOS() {
     return /CriOS/.test(navigator.userAgent)
 }
 
+function isChromeAndroidMobile() {
+    return /Mobile/.test(navigator.userAgent)
```

Comment:
This test seems very broad, are you sure there won't be any false positives?

Reply:
Stumbled into this:

![image](https://user-images.githubusercontent.com/677686/105927274-af7f5400-6011-11eb-9360-d1e1614a4291.png)

https://developer.chrome.com/docs/multidevice/user-agent/

Reply:
That's perfect 😅  I guess the function I was using as a reference didn't want to use a regex for some reason.

Reply:
Ok should be good now. Thanks guys!